dotnet/AgrirouterApiProtobufDefinitions/AgrirouterApiProtobufDefinitions/AgrirouterApiProtobufDefinitions.csproj

```diff
@@ -14,7 +14,7 @@
     </PropertyGroup>
 
     <ItemGroup>
-      <PackageReference Include="Google.Protobuf" Version="3.11.1" />
+      <PackageReference Include="Google.Protobuf" Version="3.16.1" />
```

Comment:
Currently, the latest version is 3.19.4, should we not choose that one?

https://www.nuget.org/packages/Google.Protobuf/

Reply:
Agreed and Updated